api/v1alpha3/types.go

```diff
@@ -38,6 +38,11 @@ type Network struct {
 
 // NetworkSpec specifies what the Azure networking resources should look like.
 type NetworkSpec struct {
+	// PublicIP is the public IP to attach to the Azure load balanacer.
+	// If name is not set in the spec, one will be generated.
```

Comment:
Is "If name is not set in the spec, one will be generated." accurate? I thought the name _had_ to be provided if the user wanted to pass in there own IP?

Reply:
Let's make it more explicit that this is for providing your own control plane IP that already exists